android/java/org/chromium/chrome/browser/vpn/utils/BraveVpnUtils.java

```diff
@@ -281,4 +274,23 @@ public static void showToast(String message) {
         Context context = ContextUtils.getApplicationContext();
         Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
     }
+
+    private static boolean isRegionSupported() {
+        BraveRewardsNativeWorker braveRewardsNativeWorker = BraveRewardsNativeWorker.getInstance();
+        return (braveRewardsNativeWorker != null && braveRewardsNativeWorker.IsSupported());
```

Comment:
do we support the same regions as in rewards?

Reply:
yeah we don't support VPN OFAC countries.